test/StreamableBodyTest.cpp

```diff
@@ -4,11 +4,20 @@
 
 #include <gtest/gtest.h>
 
+#include <memory>

```

Comment:
Do you still need this? Doesn't everything have the same type and no pointer now?